utils/compare_fits_dirs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Same question as above, why not using https://github.com/VERITAS-Observatory/V2DL3/blob/main/pyV2DL3/script/compareFitsFiles.py ?